graph-builder/Cargo.toml

```diff
@@ -42,8 +42,6 @@ built = "^0.3.2"
 
 [dev-dependencies]
 twoway = "^0.2"
-assert-json-diff = "1.0.0"
-test-case = "1.0.0"
```

Comment:
Does `cargo update` removes the stale entries? If not which command?

Reply:
`cargo update` doesn't alter the `Cargo.toml` at all, but only `Cargo.lock`.

Reply:
What is the command for detecting stale entries in cargo toml file?

Reply:
I'm not aware of one. I simply knew that they weren't used anymore.